src/metabase/driver.clj

```diff
@@ -43,6 +43,12 @@
      java.time.format.TextStyle/SHORT
      (java.util.Locale/getDefault))))
 
+(defn- long-timezone-name [timezone-id]
+  (let [^java.time.ZoneId zone (if (seq timezone-id)
+                                 (t/zone-id timezone-id)
+                                 (t/zone-id))]
+     zone ))
```

Comment:
This returns a `java.time.ZoneId` instance which is unexpected, I would expect this to be symmetrical to `short-timezone-name` and return a string. A simple conversion to string results in `"UTC"` on my computer, whereas `(.getDisplayName (t/zone-id) java.time.format.TextStyle/FULL (java.util.Locale/getDefault))` produces `"Coordinated Universal Time"`.

Reply:
Thank you, will take into consideration if this idea goes further.